main.py

```diff
@@ -0,0 +1,30 @@
+import requests
+from flask import Flask, render_template
+
+app = Flask(__name__)
+INDEX_HTML = "index.html"
+POST_HTML = "post.html"
+BLOG_API = "https://api.npoint.io/c790b4d5cab58020d391"
+HOME_PATH = '/'
+POST_ID_KEY = "id"
+SHOW_POST_PATH = "/path/<int:index>"
+blog_response = requests.get(BLOG_API)
```

Comment:
You'd probably want to call this from within the function.  Otherwise, it won't change results if something gets added because this is invoked at the beginning of the file.

Reply:
I created a new function to return the get request, and called it within the route functions.